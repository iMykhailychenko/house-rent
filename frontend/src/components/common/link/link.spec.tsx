import { render } from '@testing-library/react';

import Link from './link';

describe('Link component', () => {
    it('matches snapshot', () => {
        const { asFragment } = render(<Link href="/test">test</Link>);
        expect(asFragment()).toMatchSnapshot();
    });

    it('matches snapshot props', () => {
        const { asFragment } = render(
            <Link className="test" href="/test" primary title="test">
                test
            </Link>,
        );
        expect(asFragment()).toMatchSnapshot();
    });
});
