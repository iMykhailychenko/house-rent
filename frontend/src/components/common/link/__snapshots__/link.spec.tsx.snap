// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link component matches snapshot 1`] = `
<DocumentFragment>
  <a
    class="link"
    href="/test"
  >
    <span
      class="children"
    >
      test
    </span>
  </a>
</DocumentFragment>
`;

exports[`Link component matches snapshot props 1`] = `
<DocumentFragment>
  <a
    class="test link primary"
    href="/test"
    title="test"
  >
    <span
      class="children"
    >
      test
    </span>
  </a>
</DocumentFragment>
`;
